jenkins/stage-maven-release.jenkinsfile

```diff
@@ -0,0 +1,110 @@
+pipeline {
```

Comment:
Add lib so that jenkins shared libraries can be used from build repo.

```
lib = library(identifier: 'jenkins@main', retriever: modernSCM([
    $class: 'GitSCMSource',
   remote: 'https://github.com/opensearch-project/opensearch-build.git',
]))
```

Reply:
Sorry I experimented with below and worked. Mind changing to
```
library identifier: 'jenkins@main', retriever: modernSCM([
    $class: 'GitSCMSource',
   remote: 'https://github.com/opensearch-project/opensearch-build.git'])
```
Thanks!